### PR TITLE
feat(text-field): add value prefix and badge

### PR DESCRIPTION
**Pull request by pedrorebelocosta**

### CODE REVIEW

**Review by djcouto — APPROVED** [review]
Very nice work 👍

File: packages/react-ui-components/src/stories/Components/LabelsDropdown.stories.tsx
[the remark this answers is not in the document]
Comment (pedrorebelocosta) [reply packages/react-ui-components/src/stories/Components/LabelsDropdown.stories.tsx:1]: never used therefore we decided to ☠️ it

File: packages/ui-components/src/components/badge/badge.base.scss [hunk 1]
```diff
@@ -35,11 +37,13 @@
...
 
-	width: var(--badge-width);
+	min-width: var(--badge-min-width);
+	max-width: var(--badge-max-width);
 	height: var(--badge-height);
+	padding: 0 $spacing;
 	line-height: var(--badge-height);
 
-	border-radius: 100%;
+	border-radius: $spacing-2x;
```
[the remark this answers is not in the document]
Comment (pedrorebelocosta) [reply packages/ui-components/src/components/badge/badge.base.scss:46]: will change :)

File: packages/react-ui-components/src/components/SchemaForm/Templates/BaseInputTemplate/BaseInputTemplate.tsx [hunk 2]
```diff
@@ -58,6 +58,8 @@ const BaseInputTemplate = <T, S extends StrictRJSFSchema = RJSFSchema, F extends
 				useInputMask={useInputMask}
 				inputMaskRegex={inputMaskRegex}
 				value={value || value === 0 ? value : ''}
+				valuePrefix={valuePrefix}
```
[the remark this answers is not in the document]
Comment (pedrorebelocosta) [reply packages/react-ui-components/src/components/SchemaForm/Templates/BaseInputTemplate/BaseInputTemplate.tsx:61]: it's a reserved keyword :/

**Review by GoncaloCamaz — APPROVED** [review]

**Pull request merged**